docs2017/src/api/design_documents.md

```diff
@@ -504,7 +504,7 @@ because they are based on additional factors such as query parameters or the use
 The [`req`](#req) argument corresponds to that used in a [list function](#list-functions).
 
 When you have defined a show function,
-you query it by sending a `GET` request to `https://$USERNAME.cloudant.com/$DATABASE/$DESIGN_ID/_show/$SHOW_FUNCTION/$DOCUMENT_ID`,
+you query it by sending a `GET` request to `https://$ACCOUNT.cloudant.com/$DATABASE/$DESIGN_ID/_show/$SHOW_FUNCTION/$DOCUMENT_ID`,
```

Comment:
other examples suggest we're standardizing on `_design/$DDOC` rather than `$DESIGN_ID` - replace this with:
you query it by sending a `GET` request to `https://$ACCOUNT.cloudant.com/$DATABASE/_design/$DDOC/_show/$SHOW_FUNCTION/$DOCUMENT_ID`,